src/Schedule.cpp

```diff
@@ -60,6 +68,12 @@ std::string LoopLevel::func() const {
     return contents->func_name;
 }
 
+int LoopLevel::stage_index() const {
```

Comment:
There will be a nontrivial merge collision between this and https://github.com/halide/Halide/pull/2504, but I don't think that it will be hard to resolve.

Reply:
What is causing the collision?